telco-core/networking/multinetworkpolicy/multiNetworkPolicyAllowPortProtocol.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: k8s.cni.cncf.io/v1beta1
+kind: MultiNetworkPolicy
+metadata:
+  name: allow-port-and-protocol
+  annotations:
+    k8s.v1.cni.cncf.io/policy-for: <network_name>
```

Comment:
For user variable content use $network_name (ie $anything-can-go-here) 
Same comment for user variations below